qiling/loader/loader.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, ql):
         self.argv   = self.ql.argv
         self.images = []
         self.coverage_image = namedtuple('Image', 'base end path')
+        self.skip_exit_check = False
```

Comment:
Could we have a property for this?

Reply:
I know some other members are not properties since the refactor is still going on.

Reply:
Sure, I assumed this was only for user-facing properties.
(Did you guys measure the performance impact of the annotated functions vs just setting variables?)

Reply:
Okay, let me do that when I refactor this part.